test/integration/bad/new/stanc.expected

```diff
@@ -2042,13 +2042,13 @@ Expected "offset = " expression (not containing binary logical operators).
 
   $ ../../../../../install/default/bin/stanc location-scale-bad9.stan
 
-Syntax error in 'location-scale-bad9.stan', line 1, column 27 to column 28, parsing error:
+Syntax error in 'location-scale-bad9.stan', line 1, column 28 to column 29, parsing error:
    -------------------------------------------------
      1:  data { vector<multiplier = y,> x;}
-                                     ^
+                                      ^
    -------------------------------------------------
 
-Expected ">" after "multiplier =" expression.
+Expected '>' or lower expression after multiplier expression.
```

Comment:
I think this should be `'>' or offset expression`.

Reply:
Fixed. Nice catch!